web/setupLogger.ts

```diff
@@ -0,0 +1,19 @@
+import { isCommandArg } from '../common/util/basic';
+import { configure } from 'log4js';
+
+try {
+    configure({
+        appenders: {
+            out: { type: 'stdout', layout: { type: 'coloured' } },
```

Comment:
I've changed this from `stderr` to `stdout`. Please let me know if this is a problem

Reply:
What are the advantages?

Reply:
stdout is the default log stream and you could theoretically split error logs into it's own stream, so that you can separate them more easily in the command line.
But in such applications it doesn't make sense to use stderr, because all of them will be shipped to a central place and include a level tag.